File: data/data.code.golang.client/ack_role_login_ok.go
package proto

import (
	"packet"
)

type AckRoleLoginOk struct {
	Uname                    string
}

func AckRoleLoginOkDecode(pack *packet.Packet) *AckRoleLoginOk {
	ackRoleLoginOk := &AckRoleLoginOk{}

	ackRoleLoginOk.Uname = pack.ReadString()
	return ackRoleLoginOk
}

File: data/data.code.typescript.laya2.client/AckRoleLoginOk.ts
import Packet from './Packet'


export default class AckRoleLoginOk
{
	private _uname: string;


	constructor(packet?: Packet) {
		if (packet) {
			this._uname = packet.ReadString();
		}
	}


	public get uname(): string { return this._uname; }
	public set uname(value: string) { this._uname = value; }
}
